plugins/catalog-backend-module-bitbucket-server/src/lib/BitbucketServerClient.ts

```diff
@@ -81,7 +82,14 @@ export class BitbucketServerClient {
       request,
       getBitbucketServerRequestOptions(this.config),
     );
-    return response.json();
+    try {
+      const json_res = await response.json();
+      return json_res;
```

Comment:
Is there a reason not to follow camelCase here?
Also, you could omit the variable here and collapse as it was before.

Reply:
I will collapse.